src/util/DeepLinkParser.js

```diff
@@ -159,13 +160,14 @@ function parseDownloadLink(url: URL): PromotionLink {
 }
 
 /**
- * Parse a request for payment address link (BitPay).
- * TODO: Not fully spec'd yet.
+ * Parse a request for payment address link (BitWage).
  */
 function parseRequestAddress(url: URL): DeepLink {
   const query = parseQuery(url.query)
   const codesString = query.codes ?? undefined
-  const redir = query.redir ?? undefined
+
+  // Ensure redir doesn't end with a '/'
+  const redir = query.redir != null ? enforceLastChar(omitLastChar(query.redir, '/'), '?') : undefined
```

Comment:
Can't enforce adding a `?` at the end of the redir URI. The redir might already have a param. ie `https://bitwage.com/reqaddr/?requestId=123456`

In that sense, you'd need to add params to the param list and send something like

`https://bitwage.com/reqaddr/?requestId=123456&BTC=1z9823498274987`